### PR TITLE
feat(changelog): secret-manager-added-secret-types-are-available-on-cons 2024-07-25

### PR DESCRIPTION
**Pull request by ofranc**
Reporter: Cyril Petel

This PR is a new changelog contribution. It has been created automatically.

Make sure to review it carefully before merging it.

### CODE REVIEW

File: changelog/july2024/2024-07-25-secret-manager-added-secret-types-are-available-on-cons.mdx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+status: added
+author:
+    fullname: 'Join the #secret-manager channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2024-07-25
+category: security-identity
+product: secret-manager
+---
+
+Secret types are now displayed on Console for better usability of Secrets.
```
Comment (bene2k1) [inline changelog/july2024/2024-07-25-secret-manager-added-secret-types-are-available-on-cons.mdx:12]:
```suggestion
Secret types are now displayed in the Scaleway console for better usability of Secrets.
```

File: changelog/july2024/2024-07-25-secret-manager-added-secret-types-are-available-on-cons.mdx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+All secrets not explicitly tagged appear as opaque, but you can now decide to create new secrets of a specific type (among opaque, Username/password, Database Credential, SSH Key, Certificate, Key/Value), and secrets versions will be of the same type.
```
Comment (bene2k1) [inline changelog/july2024/2024-07-25-secret-manager-added-secret-types-are-available-on-cons.mdx:14]: Not sure about the uppercases here ( cc @nerda-codes )

File: changelog/july2024/2024-07-25-secret-manager-added-secret-types-are-available-on-cons.mdx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+Read [our documentation](https://www.scaleway.com/en/docs/identity-and-access-management/secret-manager/api-cli/create-secret-with-type/) for more details on how to use types in API.
```
Comment (bene2k1) [inline changelog/july2024/2024-07-25-secret-manager-added-secret-types-are-available-on-cons.mdx:16]:
```suggestion
Read [our documentation](/identity-and-access-management/secret-manager/api-cli/create-secret-with-type/) for more details on how to use types in API.
```

File: changelog/july2024/2024-07-25-secret-manager-added-secret-types-are-available-on-cons.mdx (hunk 2 above)
Comment (nerda-codes) [inline changelog/july2024/2024-07-25-secret-manager-added-secret-types-are-available-on-cons.mdx:14]:
```suggestion
All secrets not explicitly tagged appear as opaque, but you can now decide to create new secrets of a specific [type](/identity-and-access-management/secret-manager/concepts/#secret-types) (among opaque, username/password, database credentials, SSH key, SSL/TLS certificate and key/value), and secrets versions will be of the same type.
```

File: changelog/july2024/2024-07-25-secret-manager-added-secret-types-are-available-on-cons.mdx [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+title: Secret types are available on Console
```
Comment (nerda-codes) [inline changelog/july2024/2024-07-25-secret-manager-added-secret-types-are-available-on-cons.mdx:2]:
```suggestion
title: Secret types are available in the Scaleway console
```

**Review by nerda-codes — APPROVED** [review]

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**